doc/radosgw/multisite.rst

```diff
@@ -31,10 +31,13 @@ non-master zones. Following are the basic terminologies that would be used:
   is incremented on every commit operation. Every realm has an associated
   current period, holding the current state of configuration of the zonegroups
   and storage policies. Any configuration change for a non-master zone will
-  increment the period's epoch. Changing the master zone to a different zone
+  increase the period's epoch. Changing the master zone to a different zone
```

Comment:
hmm, maybe you are right. but from my point a verb `increase ` sounds reasonable here :)